LoRaEngine/test/LoRaWanNetworkServer.Test/MessageProcessorSingleGatewayTest.cs

```diff
@@ -260,87 +260,6 @@ public async Task OTAA_Unconfirmed_Message_With_FcntUp_10_Should_Send_Data_To_Io
             Assert.False(loraDevice.HasFrameCountChanges);
         }
 
-        [Fact]
-        public async Task OTAA_Unconfirmed_With_Cloud_To_Device_Message_Returns_Downstream_Message()
```

Comment:
Is the deletion of this test intentional?

Reply:
Moved to here https://github.com/Azure/iotedge-lorawan-starterkit/blob/fbe-dev-disablec2d-always2ndwindow/LoRaEngine/test/LoRaWanNetworkServer.Test/MessageProcessor_End2End_NoDep_CloudToDeviceMessage_Tests.cs